integrations/ktor/src/commonTest/kotlin/com/datadog/kmp/ktor/TracingHeaderTypeTest.kt

```diff
@@ -0,0 +1,188 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.kmp.ktor
+
+import com.datadog.kmp.ktor.internal.trace.SpanId
+import com.datadog.kmp.ktor.internal.trace.TraceId
+import io.ktor.client.request.HttpRequestBuilder
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class TracingHeaderTypeTest {
```

Comment:
could be `internal`

Reply:
Is there a benefit of doing this, given it is a `test` class?

Reply:
Optimization of build performance maybe, but it' not  a strict requirement 👍